DefensiveItem.h

```diff
@@ -8,4 +8,3 @@ struct DefensiveItem : Item
     DefensiveItem() : Item("defensive item", 3) { }
```

Comment:
Character isn't used in this header file, so you can forward declare it.
include in the cpp file.
- [ ] revise